### DISCUSSION TITLE
Add Query Load time

### DISCUSSION THREAD

**Discussion by rebase-master** [julienbourdeau/debugbar#34] [Ideas]
Hi,

It would be nice and useful to have query load times for all the queries (under the query tab) generated on a page. This would be pretty helpful in identifying and optimizing database queries for performance.

**Comment by julienbourdeau** [comment]
Yeah I agree! If you had ask me, I would have bet it was already there 🙈

**Comment by julienbourdeau** [comment]
I updated the "queries" section to add timings in v0.3.2. 
It also hides the source location by default. Thank you for feedback!

![CleanShot 2024-05-04 at 09 49 11@2x](https://github.com/julienbourdeau/debugbar/assets/1525636/6f991415-fd9c-4c0b-b737-0754e7137e87)
